code/modules/reagents/chemistry/reagents_datum.dm

```diff
@@ -37,6 +37,8 @@
 	var/taste_description = "metaphorical salt"
 	/// how quickly the addiction threshold var decays
 	var/addiction_decay_rate = 0.01
+	var/invisible_to_scanners = FALSE // If TRUE, this cannot be detected inside a body with the advanced health scanner's reagent scan
+	var/unable_to_purge = FALSE // If TRUE, this cannot be purged with mercury, calomel, or charcoal
```

Comment:
Like suggestion above, rename to `purgeable`. Also don't forget to auto doc the variables you made with `/// stuff`